docs/developer/setup.rst

```diff
@@ -179,9 +197,15 @@ Start the renku platform using helm:
     --set notebooks.jupyterhub.hub.services.gateway.oauth_redirect_uri=http://$RENKU_DOMAIN/api/auth/jupyterhub/token \
```

Comment:
I can't really make that suggestion here, but to be consistent with the docs maybe we should set the timeout to 3600s (two lines above).

Reply:
good point!